libpod/networking_pasta_linux.go

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// networking_pasta_linux.go - Start pasta(1) for user-mode connectivity
+//
+// Copyright (c) 2022 Red Hat GmbH
+// Author: Stefano Brivio <sbrivio@redhat.com>
+
+package libpod
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/sirupsen/logrus"
+)
+
+func (r *Runtime) setupPasta(ctr *Container, netns ns.NetNS) error {
+	var NoTCPInitPorts = true
+	var NoUDPInitPorts = true
+	var NoTCPNamespacePorts = true
+	var NoUDPNamespacePorts = true
+	var NoMapGW = true
+
+	path := r.config.Engine.NetworkCmdPath
```

Comment:
this needs an update in containers/comon to update the contianers.conf docs.
Although I wonder while the name is generic it would make it impossible to set a custom path for either slirp4netns or pasta at the same time, .i.e. if I have a custom slrip path set using --network pasta will just blow up unexpectedly.

Alternatively we could use the r.config.FindHelperBinary(). I think that might make more sense here.

Reply:
With `r.config.FindHelperBinary()`: would it make sense that I just call `r.config.FindHelperBinary("pasta", true)` similarly to how it's done in many other places, or should I do something different?

Should I need to add something to containers/common, how do I "coordinate" the two pull requests?

Reply:
`r.config.FindHelperBinary("pasta", true)` sounds good to me. With that I don't think we need to change containers.conf.

Generally c/common changes need to go first and are then vendored into podman, for docs changes the order it is not important.